openrest4j-api/src/main/java/com/openrest/v1_1/Location.java

```diff
@@ -59,23 +85,80 @@ public boolean equals(Object o) {
+                Objects.equals(defaultLocation, location.defaultLocation) &&
+                Objects.equals(contact, location.contact) &&
+                Objects.equals(address, location.address) &&
+                Objects.equals(timezone, location.timezone) &&
+                Objects.equals(openTimes, location.openTimes) &&
+                Objects.equals(deliveryInfos, location.deliveryInfos) &&
+                Objects.equals(deliveriesInfo, location.deliveriesInfo) &&
+                Objects.equals(orders, location.orders) &&
+                Objects.equals(reservations, location.reservations);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, name, description, defaultLocation);
+        return Objects.hash(id, name, description, defaultLocation, contact, address, timezone, openTimes, deliveryInfos, deliveriesInfo, orders, reservations);
+    }
+
+    @Override
+    public String toString() {
+        return "Location{" +
+                "id='" + id + '\'' +
+                ", name='" + name + '\'' +
+                ", description=" + description +
+                ", defaultLocation=" + defaultLocation +
+                ", contact=" + contact +
+                ", address=" + address +
+                ", timezone=" + timezone +
+                ", openTimes=" + openTimes +
+                ", deliveryInfos=" + deliveryInfos +
+                ", deliveriesInfo=" + deliveriesInfo +
+                ", orders=" + orders +
+                ", reservations=" + reservations +
+                '}';
     }
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
     public String id;
 
-    @JsonInclude(JsonInclude.Include.NON_NULL)
-    public String name;
+    /** The organization's title in various locales. */
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public LocalizedString name = LocalizedString.empty;
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
-    public String description;
+    public LocalizedString description;
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
     public Boolean defaultLocation;
+
+    /** The organization's contact. */
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Contact contact;
+
+    /** The address of this organization. */
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public Address address;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String timezone;
+
+    /** Restaurant opening times. */
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public Availability openTimes = new Availability();
+
+    /** Information regarding the different delivery destinations. */
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public List<DispatchInfo> deliveryInfos = new LinkedList<>();
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public DeliveriesInfo deliveriesInfo;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public OrdersInfo orders;
```

Comment:
I am not sure it is used or what is it for

Reply:
This is the settings of the orders, it will change to be per location in the client also.